iac/terraform/aws/auth/variables.tf

```diff
@@ -2,3 +2,8 @@ variable "environment" {
   type        = string
   description = "Deployment environment"
 }
+
+variable "lambda_archive_path" {
+  type        = string
+  description = "Path to the Lambda function zip archives"
```

Comment:
which lambda's zip archive, that should be specified.

Reply:
All lambda zips are generated in the same directory. so its value is same everywhere